### DISCUSSION TITLE
[idea] Dragula replacement candidate

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#820] [Ideas]
Hi

Even if I was able to find a [trick](https://codepen.io/GATIGA/pen/zYEypbV) to move from list A to list B (avoiding mavo's [bug](https://github.com/mavoweb/mavo/issues/738))

Referering to @LeaVerou [comment](https://github.com/mavoweb/mavo/issues/738#issuecomment-947866115) 

Could [sortable.js](https://sortablejs.github.io/Sortable/) be a good replacement candidate in the future?

Regards

**Comment by LeaVerou** [comment]
I've actually done [a fair bit of work on a fork of Dragula](https://github.com/leaverou/dragula2) recently, and @bevacqua added me as a maintainer so I could port my changes to the actual Dragula (which I haven't done yet because I got swamped with other stuff), so I'm now not very inclined to switch to another library (even though I [myself was looking for alternatives a few months ago](https://github.com/mavoweb/mavo/issues/738#issuecomment-947866115).

**Reply by LeaVerou** [reply]
Just noticed you linked to my comment as well. Why not reply there? This kind of thing is not really suitable for a Q&A

**Reply by GalinhaLX** [reply]
thank you for answer @LeaVerou , 

I post in "ideas" category , not in "Q&A" :)

I was submitting the idea because I already reply to your comment in the other message, but as there was no answer (I can understand that's difficult to follow all messages everywhere, and reply to all), that's why I post it here

I was also sending this idea to have more feedback about future: if I have to invest myself into dragula, to understand how it works, or if you intend to change to other library

**Reply by GalinhaLX** [reply]
Sorry if it causes some confusion (I 'm also new to Github :) )
